RSS Feed - Add default headers param (#37836)
demisto/content: Packs/FeedRSS/ReleaseNotes/1_0_15.md

#### Integrations

##### RSS Feed

> - Added the **Request headers** parameter.
> - The integration is now parsing the `indicator.links` value when exists in the feed.

Reviewer: ```suggestion
- Added the *Request headers* parameter.
- The integration now parses the `indicator.links` value if it exists in the feed.
```
